src/utils.cpp

```diff
@@ -385,7 +384,11 @@ std::string pyne::get_full_filepath(std::string filename) {
   // remove all extra whitespace
   filename = pyne::remove_characters(" " , filename);
   // use stdlib call
+#ifndef _WIN32
   const char* full_filepath = realpath(filename.c_str(), NULL);
+#else
+  const char* full_filepath = filename.c_str();
```

Comment:
This seems like different behavior...?

Reply:
realpath does not exist on windows.
https://www.quora.com/In-C++-why-isnt-the-function-realpath-on-Windows

Reply:
OK - but then should we change the non-windows behavior?  Do we know that `realpath()` is always necessary?

Reply:
I defaulted to use `canonical` for windows :https://en.cppreference.com/w/cpp/filesystem/canonical